TicTacToeUserInterface/UserInterface.cs

```diff
@@ -22,8 +23,15 @@ public string startNewGame()
 
             if (answer == "Y")
             {
-                var marker = chooseMarker();
-                newGame = new TicTacToe(marker);
+                Console.Write("Player One - ");
+                var markerOne = chooseMarker();
+
+                Console.Write("Player Two - ");
+                var markerTwo = chooseMarker();
+
+                adapter = new Adapter(markerOne, markerTwo);
+
+                newGame = new TicTacToe(adapter.X_MARKER, adapter.O_MARKER);

```

Comment:
I'd have to see more of how you plan to use the Adapter (I'd name it something else btw, Symbols or Options maybe) before I say I 👍 or 👎 the design. A couple things to note, you're referring to `X_MARKER` and `O_MARKER` which is confusing because you're asking the player to choose their markers. Maybe `P1_MARKER` & `P2_MARKER` or `MARKER_ONE` `MARKER_TWO`.

Keep going with this idea for now and we can refactor/delete code as you prove it out. What do you think about the design?

Side note, try to keep PRs free of commented out code.